### ISSUE TITLE
It would be better if documentation have a showcase page.

### ISSUE THREAD

**Issue by EhsanSarshar** [microsoft/react-native-windows#5592]
if it have a showcase page that shows some apps that built with react-native-windows

**Comment by chrisglein** [comment]
Agreed. I think it was meant to eventually show up here: https://microsoft.github.io/react-native-windows/resources
Not sure if we came back to that. @kikisaints ?
